tests/common/plugins/loganalyzer/loganalyzer.py

```diff
@@ -32,8 +32,9 @@ def __enter__(self):
         """
         Disable logrotate cron task / systemd timer and make sure the running logrotate is stopped.
         """
-        # Disable logrotate systemd timer
-        self.ansible_host.command("systemctl stop logrotate.timer")
+        # Disable logrotate systemd timer by best effort. The reason is that logrotate.timer service is not
+        # available in older version like 201911.
+        self.ansible_host.command("systemctl stop logrotate.timer", module_ignore_errors=True)
```

Comment:
@wangxin I was also hit by this error when running pytest test cases. I am wondering whether we can use the image version as the decision instead of ignoring errors to stop/restart logrotate timer service?

Reply:
It is not convenient to get the image version in this plugin. Need some dirty changes to have that information available. It would be cleaner to use a best effort method here.